cmd/wallet-web/src/pages/WACIIssue.vue

```diff
@@ -4,237 +4,171 @@
+            />
+          </template>
+        </credential-overview>
       </div>
     </div>
+
+    <WACI-action-buttons-container>
+      <template #leftButton>
+        <styled-button id="cancelBtn" type="outline" @click="cancel">
+          {{ t('CHAPI.Share.decline') }}
+        </styled-button>
+      </template>
+      <template #rightButton>
+        <styled-button id="storeVCBtn" type="primary" @click="save">
+          {{ t('WACI.Issue.save') }}
+        </styled-button>
+      </template>
+    </WACI-action-buttons-container>
   </div>
 </template>
 
 <script>
-import { getCredentialDisplayData, getCredentialType, getCredentialIcon, isVPType } from '@/mixins';
 import { toRaw } from 'vue';
 import { mapGetters } from 'vuex';
 import { CollectionManager, DIDComm } from '@trustbloc/wallet-sdk';
-import CustomSelect from '@/components/CustomSelect/CustomSelect.vue';
-import Spinner from '@/components/Spinner/Spinner.vue';
 import { useI18n } from 'vue-i18n';
+import { getCredentialDisplayData, getCredentialType, getCredentialIcon } from '@/mixins';
+import CustomSelect from '@/components/CustomSelect/CustomSelect.vue';
+import StyledButton from '@/components/StyledButton/StyledButton.vue';
+import CredentialOverview from '@/components/WACI/CredentialOverview.vue';
+import WACIActionButtonsContainer from '@/components/WACI/WACIActionButtonsContainer.vue';
+import WACIError from '@/components/WACI/WACIError.vue';
+import WACILoading from '@/components/WACI/WACILoading.vue';
+import WACISuccess from '@/components/WACI/WACISuccess.vue';
+import CredentialDetailsTable from '@/components/WACI/CredentialDetailsTable.vue';
 
 export default {
-  components: { CustomSelect, Spinner },
+  components: {
+    CredentialDetailsTable,
+    CredentialOverview,
+    CustomSelect,
+    StyledButton,
+    WACIActionButtonsContainer,
+    WACIError,
+    WACILoading,
+    WACISuccess,
+  },
   setup() {
-    const { t } = useI18n();
-    return { t };
+    const { t, locale } = useI18n();
+    return { t, locale };
   },
   data() {
     return {
       loading: true,
-      processedCredentials: [],
+      saving: false,
       errors: [],
       vaults: [],
       selectedVault: '',
       credentialDisplayData: '',
+      processedCredentials: [],
+      savedSuccessfully: false,
     };
   },
+  computed: {
+    showMainState() {
+      return !this.loading && !this.errors.length && !this.saving && !this.savedSuccessfully;
+    },
+    successButtonLabel() {
+      return this.redirectUrl ? this.t('WACI.ok') : this.t('WACI.Issue.viewCredential');
+    },
+  },
   created: async function () {
     this.loading = true;
     this.protocolHandler = this.$parent.protocolHandler;
     const invitation = toRaw(this.protocolHandler.message());
     const { user, token } = this.getCurrentUser().profile;
-    const credManifest = this.getCredentialManifestData();
+    this.credentialDisplayData = await this.getCredentialManifestData();
```

Comment:
any reason to put await here? it was there at first place so that this operation can go on asynchronously while we perform DIDComm operation.

At #263 we have await at line #263

Reply:
Addressed